Makefile

```diff
@@ -168,6 +171,7 @@ test-centos-7:
 	vagrant up vcentos7;vagrant ssh vcentos7
 
 clean:
+	rm -rf dist/ build/ qpc.egg-info/
```

Comment:
I think we need to change `qpc.egg-info/` to `qpc_tools.egg-info/`

Reply:
👍